packages/relic/lib/src/util/util.dart

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:collection/collection.dart';
+import 'package:relic/src/map/shelf_unmodifiable_map.dart';
+
+/// Run [callback] and capture any errors that would otherwise be top-leveled.
```

Comment:
It looks like these methods shouldn't be global, but belong to a class or possibly and extension on a class. If they are no longer used, they should be removed.